src/Microsoft.Health.Fhir.Anonymizer.Shared.Core.UnitTests/Utility/DateTimeUtilityTests.cs

```diff
@@ -45,16 +46,16 @@ public static IEnumerable<object[]> GetDateDataForDateShiftWithPrefix()
 
         public static IEnumerable<object[]> GetDateDataForDateShiftButShouldBeRedacted()
         {
-            yield return new object[] { new Date("2015-02"), new Date("2015") };
+            yield return new object[] { new Date("2015-02"), PartialDateTime.Parse("2015") };
             yield return new object[] { new Date("1925-02-07"), null };
         }
 
         public static IEnumerable<object[]> GetDateTimeDataForRedact()
         {
-            yield return new object[] { new FhirDateTime("2015"), new FhirDateTime("2015") };
```

Comment:
We should add test cases for instant types too

Reply:
Added.